osbs/build/build_requestv2.py

```diff
@@ -687,3 +697,38 @@ def render_name(self):
 
         # !IMPORTANT! can't be too long: https://github.com/openshift/origin/issues/733
         self.template['metadata']['name'] = name
+
+    def set_source_requests(self, reactor_config_data):
+        """
+        Sets default requests, only if not set from client or osbs config
+        """
+        source_container_key = 'source_container'
+        memory_request_key = 'memory_request'
+        cpu_request_key = 'cpu_request'
+
+        if source_container_key not in reactor_config_data:
+            return
+
+        cpu_request = reactor_config_data[source_container_key].get(cpu_request_key)
+        memory_request = reactor_config_data[source_container_key].get(memory_request_key)
+
+        if not (cpu_request or memory_request):
+            return
+
+        if self._resource_requests is None:
+            self._resource_requests = {}
+
+        if cpu_request:
+            self._resource_requests['cpu'] = cpu_request
+
+        if memory_request:
+            self._resource_requests['memory'] = memory_request
+
+    def set_data_from_reactor_config(self, reactor_config_data):
+        """
+        Sets data from reactor config
+        """
+        super(SourceBuildRequest, self).set_data_from_reactor_config(reactor_config_data)
+
+        if reactor_config_data:
+            self.set_source_requests(reactor_config_data)
```

Comment:
If you implement my previous comment, this should be moved to parent class

Reply:
ok